pathspider/cmd/measure.py

```diff
@@ -120,6 +125,16 @@ def _format_action(self, action):
                 parts += "\n\nSpider safely!"
             return parts
 
+    class UploadAction(argparse.Action):
+        def __call__(self, parser, args, values, option_string=None):
+            #check if --output is set
+            if args.output == '/dev/stdout':
```

Comment:
I really don't like this approach. PATHspider integrates with a number of other tools through pipes and this option breaks that. Unless you can hook the output writer to enable this function, I'd rather not have it at all and just have the separate upload command.

Reply:
For the uploader the output must be set different to the default dev/stdout.
Instead of checking if the --output is set it can either do "'if args.output == '/dev/stdout':' before calling the uploader or add a variable to the --upload command with the same function as --output (overwriting --output).
What do you think?